linera-storage/src/db_storage.rs

```diff
@@ -349,13 +349,17 @@ pub async fn list_all_chain_ids<S: AdminKeyValueStore>(
 
 /// An implementation of [`DualStoreRootKeyAssignment`] that stores the
 /// chain states into the first store.
+#[derive(Clone)]
```

Comment:
`Copy`?

Reply:
I do not see why, but why not.

Reply:
Standard procedure. The type is obviously going to be always Copy-iables. Now ofc I don't know if Clone/Copy was really needed